### ISSUE TITLE
not being able to reconnect after receiving" Error: tor: CLOCK_JUMPED"

### ISSUE THREAD

**Issue by kjozwiak** [brave/browser-laptop#14483]
### Description

When waking a machine, sometimes the Tor network won't reconnect and you'll get `Tor connection error: Tor not ready: Error: tor: CLOCK_JUMPED` in the browser console. You'll get a similar error within `tor.log`:

`Jun 20 22:56:34.000 [notice] Your system clock just jumped 109 seconds forward; assuming established circuits no longer work.`

When you run into the above situation, you'll have to restart Brave to get it Tor PT's working again.

### Steps to Reproduce

  1. launch `0.23.11` 6565c06 via the terminal/console and ensure that the Tor services starts
  2. open `Energy Saver` (macOS) and set `Turn off Display under `Battery` for like 2-3 mins
  3. ensure your mac is unplugged and wait till the display eventually turns off
  4. wait a few more minutes and wake up the machine
  5. attempt opening a Tor PT

Here's the settings I used under `Energy Saver`:

Turn display off after: 3mins

* [x] Put hard disk to sleep when possible
* [x] Slightly dim the display while on battery power
* [ ] Enable Power Nap while on batter power

**Actual result:**

<img width="506" alt="screen shot 2018-06-21 at 1 08 11 am" src="https://user-images.githubusercontent.com/2602313/41699476-07290312-74f2-11e8-8e0f-2b219201b1ee.png">

You'll get the following in the browser console:

`Tor connection error: Tor not ready: Error: tor: CLOCK_JUMPED`

You'll get the following within the `tor.log`:

`Jun 20 22:56:34.000 [notice] Your system clock just jumped 109 seconds forward; assuming established circuits no longer work.`

**Expected result:**

Should be able to reconnect to the Tor network without any issues after waking up the machine.

**Reproduces how often:**

It's not 100% reproducible and doesn't always happen but I managed to reproduce the issue twice after about 6 attempts. It might also be due to how long you keep your machine asleep before waking up and attempting to use Tor PT's.

### Brave Version

**about:brave info:**

 
```
Brave: 0.23.11 
V8: 6.7.288.46 
rev: 6565c0689f8bfbd6f4d03e2e8c68c9381da157f6 
Muon: 7.1.0 
OS Release: 17.5.0 
Update Channel: Beta 
OS Architecture: x64 
OS Platform: macOS 
Node.js: 7.9.0 
Brave Sync: v1.4.2 
libchromiumcontent: 67.0.3396.87
```

**Reproducible on current live release:**

**Comment by bsclifton** [comment]
Fixed with https://github.com/brave/browser-laptop/pull/14499

**Comment by LaurenWags** [comment]
Verified with macOS 10.12.6 using
* 0.23.14 f4da855
* Muon 7.1.1
* libchromiumcontent 67.0.3396.87

**Comment by kjozwiak** [comment]
Quick update.. I tried this a bunch of times on the weekend where I opened a few Tor PT's and closed the laptop/let it go into sleep mode. After a few hours, I opened the laptop and reconnected without any issues. Didn't run into the above issues.

Sometimes you'll receive the `Tor can't connect` modal but it only appears for 1-2 seconds before it disappears when the connection is made. Most of the time it reconnects without displaying the modal.
